.gitignore

```
@@ -0,0 +1,179 @@
+# Служебные директории macOS
+**/.DS_Store
```

```suggestion
.DS_Store
Thumbs.db
```

Если я ничего не путаю, то такого правила будет достаточно, для игнора этого файла где угодно. Заодно добавила ещё один системный файл